daemon/README

```diff
@@ -0,0 +1,193 @@
+Existing User Tracking
+======================
+
+- tracking is done by:
+  - monitoring /etc/passwd file
+
+- maintained data:
+  - minimum UID
+  - maximum UID
+  - set of valid UID values
+
+- UID added:
```

Comment:
I'm not sure if I follow how adding a user could lead into deleting data for some user...

Reply:
I meant, there could be several sequential events:
1) Remove user 100003
2) Power loss
3) Add user 100003
So some data could left in the database because of the power loss in the middle of user removal. So when a new user will be added with the same UID, some permission approvals could be inherited from the previous user instance.
That's why I though about removing user data prior to adding new users too -- just in a case of stale data.

Reply:
2.5. device boots up, sailjaild is started,  stale user data is ignored (already now) / removed (continuation task).

When/if we rely on user manager we will have all kinds of interesting synchronization gotchas, but until that I do not see problem with the above power loss scenario. And even if there would be, I do not see it as something that should block the initial implementation.